vibracode/ecommerce-website-html-css
# Files added successfully

Title:
Added content for About, Contact Us, Blog, and Shop pages; improved responsiveness using Bootstrap

Description:
-Created "About Us" page with company information and images.
-Added "Contact Us" page with a contact form and contact details.
-Developed "Blog" page featuring recent posts with images and brief descriptions.
-Created "Shop" page showcasing product listings with images, prices, and purchase options.
-Implemented Bootstrap 5 for responsive design across all pages.
-Refactored the navigation bar to use Bootstrap's navbar component for mobile responsiveness.
-Ensured that all sections (Home, Shop, Blog, About, Contact Us) are accessible and responsive on various screen sizes.


How to Test:
-Open the "index.html" file and navigate through all menu items to check if the pages load correctly.
-Verify that the layout adjusts properly on mobile, tablet, and desktop devices.
-Check that the contact form on the "Contact Us" page is visually correct.
-Confirm that the "Blog" page displays recent posts with images and descriptions.
-Ensure the "Shop" page displays product listings with proper details and "Add to Cart" functionality.


Screenshots:
Desktop view: (screenshot of the desktop layout)
![Screenshot (699)](https://github.com/user-attachments/assets/5e718fae-4a74-46bf-a948-8d0963de8e82)


Mobile view: (screenshot of the mobile layout)
![s1](https://github.com/user-attachments/assets/de71a3c6-970a-4363-b940-3ba3d467f873)
![s2](https://github.com/user-attachments/assets/e5410bb5-c501-4366-99cb-a556980de752)
![s3](https://github.com/user-attachments/assets/5be848a9-af70-4b40-b4fe-e0409f9198aa)
![s4](https://github.com/user-attachments/assets/9be49ab9-be97-40f8-bba3-ecefa394e189)

Related Issues:
Fixes : Improved website responsiveness
